specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2016-04-01/workspaces.json

```diff
@@ -88,7 +88,7 @@
         ],
         "x-ms-examples": {
           "WorkspaceGet": {
-            "$ref": "../examples/GetWorkspace.json"
+            "$ref": "../2016-04-01/examples/GetWorkspace.json"
```

Comment:
This can be simplified to ./examples/GetWorkspace.json (same applies to all these relative paths).